source/software/debuggers/index.rst

```diff
@@ -0,0 +1,281 @@
+
+Debuggers
+=========
+
+
+Debugging with DDT
+------------------
+
+Linaro DDT (formerly ARM DDT) is a powerful, easy to use graphical debugger. With Linaro DDT it is possible to debug: 
+
+- Single process and multithreaded software 
+- Parallel (MPI) Software 
+- OpenMP
+- Hybrid codes mixing paradigms such as MPI + OpenMP, or MPI + CUDA
+
+
+Linaro Forge supports:
```

Comment:
Stupid question -- what is "Linaro Forge" vs Linaro DDT?

Reply:
Linaro Forge is the company that now develops DDT.  They also use Forge as the meta package that has both DDT and MAP.